src/main.h

```diff
@@ -328,9 +329,10 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& ma
  * This does not modify the UTXO set. If pvChecks is not NULL, script checks are pushed onto it
  * instead of being performed inline.
  */
-bool ContextualCheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsViewCache &view, bool fScriptChecks,
+bool ContextualCheckInputs(const TZE& tze, const CTransaction& tx, CValidationState &state, const CCoinsViewCache &view, 
```

Comment:
Nit: The `tze` argument should not be first here; it should be placed between `txdata` and `consensusParams`.

Reply:
My rationale behind this location is that the TZE capability is context, rather than data. So I think it should go at the beginning or at the end, but not in the middle; my preference for the beginning is that it's better for partial application - not applicable here, but as a general practice context/capability parameters going first is good style.